src/core/style/_variables.scss

```diff
@@ -3,6 +3,10 @@
 
 // Typography
 $md-body-font-size-base: rem(1.400) !default;
+$font-family: Roboto, 'Helvetica Neue', sans-serif !default;
+
+// Media queries
+$md-mobile-screen: 600px;
```

Comment:
Oh, and we could also just create aliases for the devices if people wanted to consume them, e.g.,
```scss
$md-xsmall: "max-width: 600px";
$md-small-handset-portrait: "max-width: 360px";
$md-medium-handset-portrait: "max-width: 400px";
$md-large-handset-landscape: "max-width: 960px";
```